src/Model/Command/ItemPropertySetup.php

```diff
@@ -60,6 +60,7 @@ public static function set(string $propertyName): self
     protected function setPropertyName(string $propertyName): void
     {
         Assertion::typeIdentifier($propertyName);
+        Assertion::notEq($propertyName, 'item_id', 'Cannot define property with name item_id - please use different name');
```

Comment:
Dtto, maybe instead explain the item_id is set up in Matej by default and don't need to be set up explicitly.